spec/index.md

```diff
@@ -64,5 +64,5 @@ Gary Leavens, Sebastian Maneth, Erik Meijer, Klaus Ostermann, Didier
 Rémy, Mads Torgersen, and Philip Wadler have shaped the design of
 the language through lively and inspiring discussions and comments on
 previous versions of this document.  The contributors to the Scala
-mailing list have also given very useful feedback that helped us
+mailing list has also given very useful feedback that helped us
```

Comment:
The subject here is "contributors"

Reply:
There was also an agreement issue in the other PR, where I incorrectly supposed no one had time to review words at this late date.